Added definition tweaks; added fix for blank content for tooltips

choult opened the pull request:
Some tweaks we've needed to make for the main site

Commit:
Added definition tweaks; added fix for blank content for tooltips

michaldudek commented:
Pulled it, but also got a bug (and fixed it). JavaScript doesn't have a String.trim() function - Prototype adds it. I switched it to jQuery's $.trim(). So you should probably pull back ;)

choult commented:
Cool, will do.

Probably got a couple more to add in soon - mostly around fetching Learn
More content.

Chris

On 10 September 2012 18:42, Michal Dudek notifications@github.com wrote:

> Pulled it, but also got a bug (and fixed it). JavaScript doesn't have a
> String.trim() function - Prototype adds it. I switched it to jQuery's
> $.trim(). So you should probably pull back ;)
> 
> —
> Reply to this email directly or view it on GitHubhttps://github.com/datasift/editor/pull/1#issuecomment-8431266.
